openfoodfacts/images.py

```diff
@@ -254,3 +254,99 @@ def download_image(
         session=session,
         return_struct=return_struct,
     )
+
+
+def convert_to_legacy_schema(images: JSONType) -> JSONType:
```

Comment:
For reference, there are corresponding downgrade / upgrade functions in ProductOpener:
https://github.com/openfoodfacts/openfoodfacts-server/pull/11818/files#diff-b20ca17ded343222f68ac9626426f282d773142154865945210f3bd3e0d467cb